frontend-public/src/customPropTypes/mines.js

```diff
@@ -0,0 +1,28 @@
+import { PropTypes, shape, arrayOf } from "prop-types";
+import { minePermit } from "@/customPropTypes/permits";
+import { mineExpectedDocument } from "@/customPropTypes/documents";
+
+export const mine = shape({
+  guid: PropTypes.string.isRequired,
+  mine_no: PropTypes.string,
+  mine_name: PropTypes.string,
+  mine_note: PropTypes.string,
+  region_code: PropTypes.string,
+  major_mine_ind: PropTypes.bool,
+  mine_permit: arrayOf(minePermit),
+  mine_expected_documents: arrayOf(mineExpectedDocument),
+});
```

Comment:
I don't think the user is exposed to the majority of this, delete what is not needed. 

```
export const mine = shape({
  guid: PropTypes.string.isRequired,
  mine_no: PropTypes.string,
  mine_name: PropTypes.string,
  mine_expected_documents: arrayOf(mineExpectedDocument),
});
```

Reply:
Removed all of the unused props information as well as removed the permittee.